Packs/Illumio/Integrations/IllumioCore/IllumioCore_test.py

```diff
@@ -38,3 +55,224 @@ def test_test_module(requests_mock, mock_client):
     requests_mock.get("https://127.0.0.1:8443/api/v2/health",
                       status_code=200, json={})
     assert test_module(mock_client) == "ok"
+
+
+@pytest.mark.parametrize("args, err_msg", [
+    ({"port": "", "protocol": "tcp", 'start_time': '2022-07-17T12:58:33.528Z', 'end_time': "2022-07-18T12:58:33.529Z",
```

Comment:
Please clean this up to make it more readable. It's a test module, make use of whitespace.

Reply:
Yes, i'll do that